amr-wind/physics/SuperEllipseBody.cpp

```diff
@@ -0,0 +1,102 @@
+#include "amr-wind/physics/SuperEllipseBody.H"
+#include "amr-wind/CFDSim.H"
+#include "AMReX_iMultiFab.H"
+#include "AMReX_MultiFabUtil.H"
+#include "AMReX_ParmParse.H"
+#include "AMReX_ParReduce.H"
+#include "amr-wind/utilities/trig_ops.H"
+#include "amr-wind/utilities/IOManager.H"
+#include "amr-wind/utilities/io_utils.H"
+#include "amr-wind/utilities/linear_interpolation.H"
+
+namespace amr_wind::superellipsebody {
+
+namespace {
+
+//! Utility function to parse inputs and return a vector instance
```

Comment:
We already have this in BoxRefiner. I would say let's remove this and then move the other one to a utility or something.